src/content/newsletters/2024-09-30.mdx

```diff
@@ -45,6 +45,15 @@ This month we've replaced our home-grown HTTPS stack with the [curl](https://cur
 
 We now offload alpha and luminance mask rendering to the GPU if possible. This is implemented by using Skia's [SkSL shader language](https://skia.org/docs/user/sksl/) and yields a substantial speedup on websites making heavy use of masks.
 
+### Emoji support
+
+Before the fork, emoji support was implemented using custom bitmap images hand-drawn by SerenityOS contributors.
+However, it was temporarily broken when we switched to using Skia for rasterization.
+This month, emoji support has been restored, now implemented by rendering emoji glyphs extracted from a font: Apple Color Emoji on macOS and Noto Color Emoji on other platforms.
+Moreover, this support is more complete than before, as now we handle ZWJ sequences by utilizing grapheme-aware text segmentation, thanks to Tim Flynn.
```

Comment:
Just a note, this isn't actually more complete than before - we supported ZWJ sequences before the switch to Skia.

Reply:
Ah yeah that's probably true. We just supported inserting the bitmaps for ZWJ emoji.